LocalyticsGDPR/ios/LocalyticsGDPR-tvOS/Info.plist

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>CFBundleDevelopmentRegion</key>
+	<string>en</string>
+	<key>CFBundleExecutable</key>
+	<string>$(EXECUTABLE_NAME)</string>
+	<key>CFBundleIdentifier</key>
+	<string>org.reactjs.native.example.$(PRODUCT_NAME:rfc1034identifier)</string>
```

Comment:
Is the bundle Identifier prefix correct? Are we allowed to use org.reactjs?

Reply:
I didn't notice this, it's from `create-react-native-app`... I think for the purpose of the sample this is OK